generate-db.js

```diff
@@ -185,13 +203,13 @@ async function processRepo(repo, repoPath, arch) {
         if (!binMap.has(packageName)) {
           binMap.set(packageName, []);
         }
-        binMap.get(packageName).push(binary);
+        /**@type {string[]}*/ (binMap.get(packageName)).push(binary);
```

Comment:
That seems like a messy way to express that type annotation.

Reply:
IDK how to do [non-nullish asserts](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator) in JSDoc :sweat_smile: